src/resolvers/userResolver.ts

```diff
@@ -378,6 +380,12 @@ const resolvers: any = {
 
     async loginOrg(_: any, { orgInput: { name } }: any) {
       const organization: any = await Organization.findOne({ name });
+      if(organization){
+      if(organization.status == 'pending'|| organization.status == 'rejected'){
```

Comment:
Let's use the `enums` declared earlier. Something like `if (organization.status === ORG_PENDING)`